From f9b520f377816341d2e79106bfddf94c5afab390 Mon Sep 17 00:00:00 2001
From: Corey Bryant <corey.bryant@canonical.com>
Date: Tue, 9 Apr 2019 17:01:10 -0400
Subject: [PATCH] Set min version of python-cinderclient to 4.0.0

As of Stein, a minimum of python-cinderclient 4.0.0 is required to
create a volume. Cinder schema no longer accepts additional
properties on volume create that are not part of the API spec [1].

[1] https://review.openstack.org/#/c/573093/

Change-Id: I0fc4d5db39080985b471fe2465b7078e37417e4d
Closes-Bug: #1824017
---
 lower-constraints.txt | 2 +-
 requirements.txt      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lower-constraints.txt b/lower-constraints.txt
index 471d3bf05..23c13a2f0 100644
--- a/lower-constraints.txt
+++ b/lower-constraints.txt
@@ -92,7 +92,7 @@ pyOpenSSL==17.1.0
 pyparsing==2.1.0
 pyperclip==1.5.27
 pyScss==1.3.4
-python-cinderclient==3.3.0
+python-cinderclient==4.0.0
 python-dateutil==2.5.3
 python-glanceclient==2.8.0
 python-keystoneclient==3.15.0
diff --git a/requirements.txt b/requirements.txt
index 336f62fa6..215719ad5 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -31,7 +31,7 @@ osprofiler>=2.3.0 # Apache-2.0
 Pint>=0.5 # BSD
 pymongo!=3.1,>=3.0.2 # Apache-2.0
 pyScss!=1.3.5,>=1.3.4 # MIT License
-python-cinderclient>=3.3.0 # Apache-2.0
+python-cinderclient>=4.0.0 # Apache-2.0
 python-glanceclient>=2.8.0 # Apache-2.0
 python-keystoneclient>=3.15.0 # Apache-2.0
 python-neutronclient>=6.7.0 # Apache-2.0
-- 
2.20.1

